apps/feedback/lib/mailer.ex

```diff
@@ -5,10 +5,10 @@ defmodule Feedback.Mailer do
 
   @spec send_heat_ticket(Feedback.Message.t(), [map()]) :: {:ok, any} | {:error, any}
   def send_heat_ticket(message, photo_info) do
-    request_response = if message.request_response, do: "Yes", else: "No"
+    no_request_response = if message.no_request_response, do: "No", else: "Yes"
 
     _ =
-      if message.request_response do
+      if message.no_request_response do
```

Comment:
Does this condition need to be reversed? Or maybe even better just check for `email` instead of `no_request_response`?

Reply:
Why change it so much? Technically on the front end it's possible to have email be populated either way (e.g. if you fill it, then change your mind and hide the contact info, it doesn't automatically clear). So some additional logic would need to handle this check properly...

Reply:
Hm it does look it needs to be reversed though!

Reply:
Yeah, sorry that was confusing. I was wondering if it should be conditional on email just because as you say email could be set regardless of this flag. But whichever seems best to you is fine by me.

Reply:
I've reversed it